tests/mocha/event_procedure_change_return_test.js

```diff
@@ -180,4 +180,18 @@ suite('Procedure Change Return Event', function() {
           });
     });
   });
+
+  suite.skip('serialization', function() {
```

Comment:
Is this `skip` just because the code being tested has not yet been written?

(Here and various places below.)

Reply:
Yes indeedly